devtools/conda-envs/test.yml

```diff
@@ -7,8 +7,8 @@ dependencies:
   - cuda-version >=12
 
   # alchemiscale dependencies
-  - gufe=1.3.0
-  - openfe=1.4.0
+  - gufe=1.6.0
+  - openfe=1.6.0
```

Comment:
Not necessary for this environment, but just an FYI that we'll need openff-toolkit pinned at 0.17.0 in the runtime environment.

Reply:
Thank you! Yes, this is just the test env. I'll be pinning explicitly in our `alchemiscale.org-deployment` envs.